src/components/common/Dialogs/TransferFundsDialog/TransferFundsDialog.tsx

```diff
@@ -98,18 +50,25 @@ const TransferFundsDialog = ({
     withMeta({ navigate }),
   );
 
+  const defaultFromDomainId = selectedDomainId || Id.RootDomain;
+  const defaultToDomainId =
+    Number(
+      domainOptions.find((option) => option.value !== defaultFromDomainId)
+        ?.value,
+    ) || Id.RootDomain;
+
+  const validationSchema = getValidationSchema(colony);
+
+  type FormValues = InferType<typeof validationSchema>;
```

Comment:
You can still define this type outside the component with: `InferType<ReturnType<typeof getValidationSchema>>`

I think that would be preferable, if only for consistency